materialize/resources/resource_view.go

```diff
@@ -0,0 +1,285 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmoiron/sqlx"
+)
+
+var viewSchema = map[string]*schema.Schema{
+	"name": {
+		Description: "The identifier for the view.",
+		Type:        schema.TypeString,
+		Required:    true,
+	},
+	"schema_name": {
+		Description: "The identifier for the view schema.",
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "public",
+		ForceNew:    true,
+	},
+	"database_name": {
+		Description: "The identifier for the view database.",
+		Type:        schema.TypeString,
+		Optional:    true,
+		Default:     "materialize",
+		ForceNew:    true,
+	},
+	"qualified_name": {
+		Description: "The fully qualified name of the view.",
+		Type:        schema.TypeString,
+		Computed:    true,
+	},
+	"temporary": {
+		Description:   "Mark the view as temporary.",
+		Type:          schema.TypeBool,
+		Optional:      true,
+		ForceNew:      true,
+		ConflictsWith: []string{"or_replace"},
+	},
+	"if_not_exists": {
```

Comment:
I don't think we need a `if_not_exists` or `or_replace` fields (the same also probably is true for `temporary` though I did include it in my table PR yesterday). The user shouldn't need to set these since those configurations should be managed by the Terraform provider and state.

Reply:
good points! I've removed this for the view resource, I will remove that for the tables in the next PR for #54